docusaurus/sidebars-ios.json

```diff
@@ -104,6 +104,7 @@
     {
       "State & Offline": [
         "client/llc-client-overview",
```

Comment:
probably makes sense to update this page as well, referencing the new state as an alternative.